spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaProperties.java

```diff
@@ -631,6 +651,137 @@ public void setFailFast(boolean failFast) {
 
 	}
 
+	/**
+	 * High (and some medium) priority Streams properties and a general properties bucket.
+	 */
+	public static class Streams {
+
+		private final Ssl ssl = new Ssl();
+
+		/**
+		 * Kafka streams application.id property; default spring.application.name.
+		 */
+		private String applicationId = "${spring.application.name}";
+
+		/**
+		 * Whether or not to auto-start the streams factory bean; default false.
```

Comment:
We don't add default values to descriptions.

Reply:
Since autoStartup is usually true by default; I thought it was important to add it here. The reason it's false is I don't want to start the factory bean for an existing Boot app that uses streams but doesn't use the auto configured infrastructure (start connects to Kafka).

I did mention it in the docs, however, so I can remove it here.

Reply:
The point I was trying to make is that the metadata will already state the default is `false` (as inferred by the default value of the field). So your IDE will tell you it's false and having that in text is a duplicate that could potentially get out of sync.

Does that make sense?